sections/infrastructure.include

```diff
@@ -3851,7 +3851,7 @@ It covers
    </ol>
   </div>
 
-<h4 id="nonce-attributes">Nonce attributes</h4>
+<h4 id="nonce-attributes"><dfn>Nonce</dfn> attributes</h4>
```

Comment:
Isn't nonce defined just below? Why the extra `dfn`?

Reply:
If I don't add that dfn, I get `LINK ERROR: No 'dfn' refs found for 'nonce'.<a data-link-type="dfn" data-lt="nonce">nonce</a>`
This may be down to me not fully understanding bikeshed cross references.